### PR TITLE
tutorial 1

### LINKED ISSUE #17: tutorial 1
Dedicated issue for Tutorial 1 (see #7).
- G36 4.2 VAVR is probably the best candidate. https://github.com/open223/models.open223.info/blob/add/g36-4.2/models/g36-extensions-vav-reheat.ttl

## Model Loading

1. Parsing an existing model (fetch from a raw GitHub url in the models.open223.info?)
   *Describing the model and linking to relevant classes, e.g. https://explore.open223.info/brick/Variable_Air_Volume_Box_With_Reheat.html.*
2. A couple/few basic operations on the model. 
   *For example, performing a query using RDFLib *and* https://query.open223.info/.*

### PR DESCRIPTION
**Pull request by MatthewSteen**
Close #17.

### CODE REVIEW

File: tutorials/model_exploration.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+First, we'll create a new empty graph then parse (load) an existing graph into it using the Python RDFLib library.
+
+```{code-cell}
+from rdflib import Graph
+
+# Create a Graph
+g = Graph()
+
+# Parse in an RDF file hosted on the Internet
+g.parse("https://raw.githubusercontent.com/open223/models.open223.info/main/models/guideline36-2021-4-1.ttl", format="ttl")
```
Comment (gtfierro) [inline tutorials/model_exploration.md:35]: You can probably replace the URL with 'https://models.open223.info/guideline36-2021-4-1.ttl' now. I tested this locally and it works fine

**Review by gtfierro — APPROVED** [review]
One small change but this look great!

File: tutorials/model_exploration.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Model Exploration
```
Comment (lazlop) [inline tutorials/model_exploration.md:14]: I'm not sure this tutorial really tells you how to 'explore' a model. Maybe it should just be "Loading and Querying a Model"

Comment (lazlop) [reply tutorials/model_exploration.md:14]: I'll retract this comment, the learning objectives make it obvious enough what this is going to show!

**Review by lazlop — COMMENTED** [review]
Seems good! I think we discussed having some content in this about query.223.  In the querying section I might just add a short paragraph saying that you can load this model into query.223 for an alternative querying interface.

File: tutorials/model_exploration.md (hunk 2 above)
Comment (MatthewSteen) [reply tutorials/model_exploration.md:14]: Thanks. I generally try to keep the titles as short as possible.

**Pull request merged**